components/micro-gateway-core/src/main/ballerina/gateway/constants/constants.bal

```diff
@@ -246,6 +249,21 @@
 @Description { value: "Set of filters to be enabled"}
 @final public string FILTERS = "filters";
 
+@Description { value: "Mutual SSL related configs"}
+@final public string MTSL_CONF_INSTANCE_ID = "mtslConfig";
+@Description { value: "MutualSSL protocol name"}
+@final public string MTSL_CONF_PROTOCOL_NAME = "protocolName";
+@Description { value: "MutualSSL protocol versions"}
+@final public string MTSL_CONF_PROTOCOL_VERSIONS = "protocolVersions";
+@Description { value: "MutualSSL ciphers"}
+@final public string MTSL_CONF_CIPHERS = "ciphers";
+@Description { value: "MutualSSL Verification"}
+@final public string MTSL_CONF_SSLVERIFYCLIENT = "sslVerifyClient";
+@Description { value: "MutualSSL Enabling"}
+@final public string MTSL_CONF_ENABLE = "MTSLEnable";
+
+
```

Comment:
Please do not add new lines unnecessarily. Shall we check all the places and remove additional new lines?